src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,546 @@
+---
+title: Monitor your .NET MAUI mobile application
+tags:
+    - .NET MAUI
+    - Mobile monitoring
+    - Hybrid framework
+    - Get started
+metaDescription: "New Relic's .NET MAUI agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
```

Comment:
we use the word 'update' and not 'upgrade', for general description of updating agents and integrations (think it's in style guide)